src/Microsoft.DocAsCode.Plugins/DependencyType.cs

```diff
@@ -24,5 +25,27 @@ public class DependencyType
         /// the build phase that this type of dependency could have an effect on. [TO-DO]: remove the nullable ? when old value is overwritten
         /// </summary>
         public BuildPhase? Phase { get; set; }
+
+        /// <summary>
+        /// the transitivity of the dependency. [TO-DO]: remove the nullable ? when old value is overwritten
```

Comment:
>[TO-DO]: remove the nullable ? when old value is overwritten [](start = 48, length = 60)

For current behavior, if we assign default value to None, is it OK? So we don't need to add the nullable paramter.